kuma/users/newsletter/sendinblue.py

```diff
@@ -2,8 +2,8 @@
 
 from kuma.core.utils import requests_retry_session
 
-
 API_URL = "https://api.sendinblue.com/v3/"
+LIST_ID = settings.SENDINBLUE_LIST_ID
```

Comment:
I don't see the point in this. 
Also, it's generally better to "delay" the use of `settings.` until it's needed. It's probably not applicable here but if you don't do that you can write a test like this:

```python
def test_something(settings):
    settings.SENDINBLUE_LIST_ID = 'test'
    ...
```

Reply:
I like that it defers the knowledge of where the list ID comes from to the sendinblue module. Couldn't you also write:

```python
def test_something(settings):
    sendinblue.LIST_ID = 'test'
    ...
```

There's a textual aesthetic thing here, that I also think it reads better, so maybe that is biasing me.

Reply:
oh no I guess it would be more like:

```python
@patch('sendinblue.LIST_ID', 'test')
def test_something():
  ...
```

Reply:
Yeah, you could and it would probably work. But sticking to the convention of using `settings` is generally better. The `settings` pytest fixture is great because it's guaranteed to not leak across different tests and now you don't need to know exactly what the `mock.patch` *path* is. 

All in all, it's fine. Just a nit. Or perhaps a recommendation from my experience.